docs/conf.py

```diff
@@ -54,9 +54,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0'
+version = u'3.0'
 # The full version, including alpha/beta/rc tags.
-release = u'1.0'
+release = u'3.7.0'
```

Comment:
Same again as above, the version should be incremented independently of the products using the definition. 
```suggestion
release = u'2.0.0'
```

Reply:
I'll change these. However, this is where the defining this outside of runner actually causes more confusion than adding benefit. Admittedly its not a runner API but this interface is undoubtably bound to a specific version of runner.